build/updater.sh

```diff
@@ -2,8 +2,10 @@
 cd /data/project/query-builder-test/query-builder
 git reset --hard origin/master
 git pull
+export NODE_ENV=production
 npm install npm@6.14.8
 ./node_modules/.bin/npm install
+source env.production
```

Comment:
From the page that you linked above: is it possible that env.production is picked up automatically in the build step?

Could you verify that? If so, then we don't need that line here after all.

Reply:
it seems to be the case.

> Env Loading Priorities

An env file for a specific mode (e.g. .env.production) will take higher priority than a generic one (e.g. .env).

In addition, environment variables that already exist when Vue CLI is executed have the highest priority and will not be overwritten by .env files.

.env files are loaded at the start of vue-cli-service. Restart the service after making changes.